Kanban_doreenbinwi@yahoo.com.md

```diff
@@ -0,0 +1,75 @@
+# Topic: kanban
+
+**Aurthur**: Che Doreen
+
+**QAs Total**: 3
+
+---
+
+## Q1: What are some of the importance of WIP in kanban ?
+
+**Difficulty:** `mid`
+
+**Source:**
+
+https://www.atlanssian.com/agile/kanban/wip-limits 
+
+**Answer:**
+
+WIP means Work In Progress. This is a tool used in kanban to set limits of maximum work that can exist in each status of workflow. It is a valuable tool to Kanban because:
+
+*  It improves throughput and reduces the amount of work "nearly done", encourages the culture of "done" by forcing the team to focus on a small set of tasks. More importantly WIP makes sure blockers and bottlenecks are visible.Teams can swarm around blocking issues to get them understood, implemented and resolved where there is a clear indicator of what existing work is causing the bottleneck. These bebefits guarantee that increments of value are delivered to customers sooner.
+
+* Secondly, multitasking is deceptively time intensive. During development its easy to think "oh will pause on this one issue while I begin another" having two issues open means context switching between two different things. Ramping down on one issue and up on another isn't free-it takes time and degrades focus. It's almost always better to work through the original issue rather than starting and not completing new work. In other words, WIP limits discourages us from impeding our own workflow.
+
+* Finlly, WIP limits point out areas of chronic idleness or overload. They help the team see inefficiencies in the entire process rather than just the area in which they work.
+
+---
+
+## Q2: What is lead time and cycle time ? How can they be used in Kanban?
+
+**Difficulty:** `Senior`
+
+**Source:**
+
+https://teamhood.com/kanban-/lead-and-cycletime-kanban-metrics/
```

Comment:
Hi, this link appears to be broken. Can we fix it?

Reply:
Hi, yes I can am looking in to it.
Thanks